java/lib/ghsl/CommandInjectionRuntimeExec.qll

```diff
@@ -4,124 +4,111 @@ import semmle.code.java.dataflow.DataFlow
 private import semmle.code.java.dataflow.TaintTracking
 import semmle.code.java.dataflow.FlowSources
 
+abstract class Source extends DataFlow::Node {
+  Source() { this = this }
```

Comment:
Is this charpred needed?

Reply:
What classes extend it?

Reply:
I didn't add this and I honestly not sure how this impacts the query